src/libcalamaresui/utils/Paste.cpp

```diff
@@ -15,14 +15,17 @@
 #include <QRegularExpression>
 #include <QTcpSocket>
 #include <QUrl>
+#include <QClipboard>
+#include <QApplication>
 
 namespace CalamaresUtils
 {
 
 QString
 sendLogToPastebin( QObject* parent, const QString& ficheHost, quint16 fichePort )
 {
-    QString pasteUrlFmt = parent->tr( "Install log posted to:\n%1" );
+    QString pasteUrlFmt = parent->tr( "Install log posted to\n\n%1\n\nLink copied to clipboard" );
```

Comment:
this is a new or modified translation string - a new translation entry should be added, or the existing one modified - should that be done in the source tree, or is it an admin task via the translations service?

Reply:
I don't have an idea regarding how translation is handled. But would be good if this piece is brought into focus of translators